src/locales/it_IT/it_IT.json

```diff
@@ -46,6 +46,14 @@
         "subtitle": "Ora puoi tornare al tuo computer per continuare",
         "title": "Caricamenti riusciti"
     },
+    "cross_device_session_linked": {
+        "button_primary": "Continua",
+        "info": "Doppio controllo",
```

Comment:
This does not really make sense in Italian. It should be "Assicurati di"

Reply:
@raemarielee FYI ^